CHANGELOG.md

```diff
@@ -32,6 +32,13 @@ Changelog
 
 * Deprecated methods have been removed.
 
+### Rule matcher
+
+* **BC break:** The `match_response` and `additional_cacheable_status` 
+  configuration parameters were removed for individual match rules. 
+  
+* Cacheable status codes are now configured globally (`cacheable.response`).
```

Comment:
we should also mention that the signature of RuleMatcher changed (or that its replaced by RequestResponseMapper or whatever if we decide to rename it)